tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 from bal_tools.ecosystem import Aura
 from bal_tools.safe_tx_builder import SafeTxBuilder
 
-exempt_chains = ["fantom", "mode", "linea", "fraxtal"]
+exempt_chains = ["fantom", "mode", "linea", "fraxtal", "goerli"]
```

Comment:
We actually have subgraphs on fraxtal and mode now if you want to keep going, it's not part of this PR really though.

Reply:
https://github.com/BalancerMaxis/bal_tools/issues/19